examples/example_annotations/browse_annotations.py

```diff
@@ -0,0 +1,199 @@
+                if py_version == 2:
+                    row = [item.decode("utf8") for item in row]
+                # write header row. assumes first row in csv contains header
+                if rownum == 0:
+                    html += "<thead>"
+                    html += "<tr class='w3-green'>"  # write <tr> tag
+                    for column in row:
+                        html += '<th>' + column + '</th>'
+                    html += '<th>' + "Annotation" + '</th>'
+                    html += '</tr>\n'
+                    html += "</thead>"
+
+                # write all other rows
+                else:
+                    mdb_key = '\t'.join([row[k] for k in self.key])
+                    if mdb_key in annotation_dict:
+                        mdb_value = int(annotation_dict[mdb_key])
+                        html += '<tr id="tr_{}" bgcolor="#99ff99">'.format(mdb_key)
+                    else:
+                        mdb_value = 0
+                        html += '<tr id="tr_{}" bgcolor="#ffffff">'.format(mdb_key)
+                    print("mdb_value", mdb_value)
+                    for i_r, column in enumerate(row):
+                        html += '<td>' + column + '</td>'
+                    html += '<td><select id="{}" annotations_file="{}" onchange="annotate(this);">' \
+                            '<option value="0" {}>-</option>' \
+                            '<option value="1" {}>1: {}</option>' \
+                            '<option value="2" {}>2: {}</option>' \
+                            '<option value="3" {}>3: {}</option>' \
+                            '<option value="4" {}>4: {}</option>' \
+                            '</td>'.format(mdb_key, self.ANNOTATION_FILE,
+                                           "selected" if mdb_value == 0 else "",
+                                           "selected" if mdb_value == 1 else "", self.classes_names[1],
+                                           "selected" if mdb_value == 2 else "", self.classes_names[2],
+                                           "selected" if mdb_value == 3 else "", self.classes_names[3],
+                                           "selected" if mdb_value == 4 else "", self.classes_names[4])
+
+                    html += '</tr>\n'
+
+                # increment row count
+                rownum += 1
+                if self.max_rows is not None and rownum >= self.max_rows:
+                    html += "<tr><td colspan='{}'><center><i>following rows hidden</i></center></td></tr>\n"\
+                        .format(len(row))
+                    break
+
+            html += '</table>\n'
+        return html
+
+    def to_html(self, file_name):
+        pass
+
+
+CLASSES_NAMES = {
+    1: "One",
+    2: "Two",
+    3: "Three",
+    4: "Four",
+}
+
+@click.command()
+@click.option('--port', '-p', default=8111, help='Port')
+def main(port):
+    chain = [
+        FlexpInfoToHtml(log_file_name="log.txt"),
```

Comment:
What if we keep only `CsvToHtmlEditable` module in example? There is nothing to show and example would be cleaner.

Reply:
Agree